### DISCUSSION TITLE
GNOME 46 supported yet?

### DISCUSSION THREAD

**Discussion by harshkushwaah** [andyholmes/gnome-shell-extension-gsconnect#1769] [Q&A]
I recently updated my gnome to v46 and with many other extensions GSconnect have also stopped working, is there any workaround to make it work, or we have to wait?

**Comment by ferdnyc** [comment]
GNOME 46 support was fixed as of some very recent PRs, the code in the repo should (hopefully) be <s>fully</s> compatible though there hasn't been a release yet.

Options would be:

1. Check out https://github.com/GSConnect/gnome-shell-extension-gsconnect and build/install the extension yourself (see [Install from Git](https://github.com/GSConnect/gnome-shell-extension-gsconnect/wiki/Installation#install-from-git) in the wiki)
2. Install the latest nightly build (see [Install from Nightly Build](https://github.com/GSConnect/gnome-shell-extension-gsconnect/wiki/Installation#install-from-nightly-build) in the wiki)
3. ...Wait?

Taking either option 1 or 2 would mean you're running minimally-tested code, but if it doesn't work at all now then it can't possibly be any **more** broken.

(**Edit:** I walked back "fully compatible" as we don't have enough testing to make that claim.)
